lib/index.js

```diff
@@ -131,7 +144,7 @@ function init(callback) {
   }
 
   async.auto({
-    initDatabase: callback => callbackify(bedrock.runOnceAsync)(
+    initDatabase: callback => callbackify(bedrock.runOnce)(
```

Comment:
This change means the bedrock peer dependency needs to be bumped to at least ^3.0.0.

Reply:
I changed it to `3.x` here: https://github.com/digitalbazaar/bedrock-mongodb/pull/33/commits/4a8f8e5f397cf3594a3c9417b7dfbfed69baae2d

is that a legit peerDep semantic version option? I know that `^3.0.0` means any minor release of 3, but I saw 3.x in `bedrock-vc-issuer` so I used it.

Reply:
They expand the same:
https://www.npmjs.com/package/semver#x-ranges-12x-1x-12-https://www.npmjs.com/package/semver#caret-ranges-123-025-004